src/plugins/input_control_vis/public/vis_controller.tsx

```diff
@@ -24,26 +24,25 @@ import { ListControl } from './control/list_control_factory';
 import { InputControlVisDependencies } from './plugin';
 import { InputControlVisParams } from './types';
 
-export type InputControlVisControllerType = InstanceType<
-  ReturnType<typeof createInputControlVisController>
->;
+export type InputControlVisControllerType = ReturnType<typeof createInputControlVisController>;
 
 export const createInputControlVisController = (
   deps: InputControlVisDependencies,
-  handlers: IInterpreterRenderHandlers
+  handlers: IInterpreterRenderHandlers,
+  el: Element
 ) => {
-  return class InputControlVisController {
-    private I18nContext?: I18nStart['Context'];
-    private _isLoaded = false;
+  let I18nContext: I18nStart['Context'] | undefined;
+  let isLoaded = false;
```

Comment:
These changes were necessary because this project wasn't being built in the composite build previously and private properties on class expressions aren't compatible. Functionality is the same but avoid the type errors